src/test/java/klerer/gameoflife/GameOfLifeTest.java

```diff
@@ -15,7 +15,7 @@ public void testNextGeneration() {
                 {0, 1, 0}
         };
 
-        GameOfLife gameOfLife = new GameOfLife(initialGrid);
+        GameOfLife gameOfLife = new GameOfLife(3, 3);
```

Comment:
You can look at your changes here and understand why the code doesn't work.

This all comes down to understanding what your code actually does.